htdocs/expedition/list.php

```diff
@@ -249,8 +249,8 @@
 if ($search_sale > 0 || (!$user->rights->societe->client->voir && !$socid)) $sql .= ", ".MAIN_DB_PREFIX."societe_commerciaux as sc";
 if ($search_user > 0)
 {
-	$sql .= ", ".MAIN_DB_PREFIX."element_contact as ec";
-	$sql .= ", ".MAIN_DB_PREFIX."c_type_contact as tc";
+	$sql .= " LEFT JOIN  ".MAIN_DB_PREFIX."element_contact as ec ON u.rowid = ec.element_id";
```

Comment:
Note sure this is a good fix.
I did not find "search_user" at all, any where in the where.
So i think we missed instead a criteria where on search_user somewhere.

Reply:
I added "search_user" in where condition.

Reply:
So there is no reason to use left join on element_contact. Exact join (inner join) should be better for performance when search_user is set.
Also I did not find any use of table alias "tc", so left join on this table may be removed, don't you think so ?

Reply:
I agree and it’s done !